src/components/badges/blink/blink.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { FC } from 'react';
+import { FlexProps, Flex } from 'rebass';
+import { Labeling } from '../../..';
+// Styles
+import styles from './blink.styles';
+
+export interface BlinkProps extends Omit<FlexProps, 'css'> {
+  text?: string;
+  variant?: 'green' | 'red' | 'gray';
+}
+
+const Blink: FC<BlinkProps> = ({
+  variant = 'gray',
+  text,
+  ...props
+}: BlinkProps) => (
+  <Flex alignItems="center">
+    <Flex
+      className="dot"
+      tx="variants.badges.blink"
+      variant={variant}
+      {...props}
+      as="span"
+      sx={styles}
+    />
+    {text && (
+      <Labeling bold gray ml="8px">
```

Comment:
do we mean to always use this component with exactly the same `Labeling`?
Guess this is a question of whether the blinking indicator itself is THE piece we are building or we want it always to come with the `Labeling` there.

And since `text` is optional, I assume that this could be combined with many other typography styles.
Should we then just accept `children` and let the end-user decide what our text is gonna look like?
But then it's making things more complicated, as the indicator is of fixed size.
Gosh
  
 Oh just nevermind :D
 Guess this is good. Not deleting this just in case

Reply:
This batch was not meant to have different colors or styles. I just added some variants in case they were useful. @giardiv might want to say something here, since I followed his design